next.config.js

```diff
@@ -0,0 +1,8 @@
+const nextConfig = {
+    i18n: {
+      locales: ["en"],
+      defaultLocale: "en",
+    },
+};
+  
+module.exports = nextConfig
```

Comment:
Before you merge this, would you mind adding a blank line at the end of this file? Thanks!